src/user-management/user-management.spec.ts

```diff
@@ -882,6 +883,19 @@ describe('UserManagement', () => {
     });
   });
 
+  describe('findInvitationByToken', () => {
+    it('sends a find invitation by token request', async () => {
+      fetchOnce(invitationFixture);
+      const invitation = await workos.userManagement.findInvitationByToken(
+        invitationToken,
+      );
+      expect(fetchURL()).toContain(
+        `/user_management/invitations/by_token/${invitationToken}`,
+      );
+      expect(invitation).toMatchObject({});
```

Comment:
Should we match a more specific deserialized object here?

Reply:
Good catch - I added a check that the invitation has the right token. I also updated the `getInvitation` test to check that the invitation has the right ID in that case. But we can also check against the entire object from the fixture if that's preferred.